src/inmanta/parser/plyInmantaLex.py

```diff
@@ -65,7 +65,7 @@
 
 
 def t_RSTRING(t: lex.LexToken) -> lex.LexToken:  # noqa: N802
-    r"r(\"([^\\\"]|\\.)*\")|r(\'([^\\\']|\\.)*\')"
+    r"r(\"([^\\\"\n\r]|\\.)*\")|r(\'([^\\\'\n\r]|\\.)*\')"
```

Comment:
why `\r`? at this point there should be no '\r`

Reply:
Though it doesn't hurt to have it here either and the fact that there should be no `\r` is an implementation detail more than an invariant.

Reply:
it is rather important, because the line numbering code also only does '\n'